### DISCUSSION TITLE
Nextcloud Talk - File picker does not pick file correctly

### DISCUSSION THREAD

**Discussion by ebildebil** [nextcloud/all-in-one#3551] [Questions] [answered]
Hi,

I am not sure where to post this so starting here.

Today I was trying to share a file in a Talk conversation. (using the '/' --> Files...)
The link that got inserted after selecting the file i wanted to share was:
https://domain.tld/f/%7BfileId%7D

This gave an error - File cannot be accessed.

Then I actually went to the Files app, and picked the local share link there which was:
https://domain.tld/f/10200006

I pasted this in chat and this worked. It seems file picker added a place holder instead of the actual file id?
Is there something I can do to fix it?

Not sure if relevant but I also see the below errors in my logs:

1.
thiagoalessio\TesseractOCR\TesseractNotFoundException: Error! The command "tesseract" was not found. Make sure you have Tesseract OCR installed on your system: https://github.com/tesseract-ocr/tesseract The current $PATH is /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

2.
richdocuments | GuzzleHttp\Exception\ClientException: Client error: `POST https://domain.tld//lool/convert-to/png` resulted in a `404 Not Found` response: <!DOCTYPE html> <html class="ng-csp" data-placeholder-focus="false" lang="en" data-locale="en" translate="no" > <head (truncated...)

**Comment by szaimen** [comment, marked as answer]
> Nextcloud Talk - File picker does not pick file correctly

This needs to be reported to and fixed in nextcloud/server

---

### The following issues are indeed unrelated

> thiagoalessio\TesseractOCR\TesseractNotFoundException: Error! The command "tesseract" was not found. Make sure you have Tesseract OCR installed on your system: [tesseract-ocr/tesseract](https://github.com/tesseract-ocr/tesseract) The current $PATH is /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

I guess you need to add it via https://github.com/nextcloud/all-in-one#how-to-add-os-packages-permanently-to-the-nextcloud-container

> richdocuments | GuzzleHttp\Exception\ClientException: Client error: `POST https://domain.tld//lool/convert-to/png` resulted in a `404 Not Found` response: <head (truncated...)

This either needs to be fixed in nextcloud/richdocuments or probably rather in CollaboraOnline/online
